apps/emqx_conf/etc/emqx_conf.conf

```diff
@@ -1,11 +1,14 @@
 ## NOTE:
-## Configs in this file might be overridden by:
-## 1. Environment variables which start with 'EMQX_' prefix
-## 2. File $EMQX_NODE__DATA_DIR/configs/cluster-override.conf
-## 3. File $EMQX_NODE__DATA_DIR/configs/local-override.conf
-##
-## The *-override.conf files are overwritten at runtime when changes
-## are made from EMQX dashboard UI, management HTTP API, or CLI.
+## The order of priority for configuration is:
+## environment variables, 'etc/emqx.conf', and 'data/configs/cluster.hocon'.
+## 1. Settings in environment variables starting with 'EMQX_' are given the highest priority.
+## 2. Configuration settings in the 'etc/emqx.conf' file may be overwritten by environment variables.
+## 3. Changes made through the EMQX dashboard UI, management HTTP API, or CLI
+##    overwrite the 'data/configs/cluster.hocon' file at runtime.
+## 4. If you modify a configuration setting using the API, the change takes effect immediately.
+## 5. However, if the same setting is configured differently in the 'etc/emqx.conf' file,
+##    'etc/emqx.conf' takes priority after a reboot.
```

Comment:
we can use the same text as in the pr sent to emqx-docs.git.

Reply:
Done